### PR TITLE
RHDEVDOCS-4437 - 5.6 Logging Release Notes

### PR DESCRIPTION
**Pull request by libander**
Version(s): 4.11+

Issue: [RHDEVDOCS-4437](https://issues.redhat.com//browse/RHDEVDOCS-4437), [RHDEVDOCS-4419](https://issues.redhat.com/browse/RHDEVDOCS-4419)

Link to docs preview: https://drive.google.com/file/d/1JqPbFD9Jb9NMyQ83_0r0nbGhYgQXr81Q/view?usp=sharing

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by libander** [comment]
Note to reviewers: 
Currently review is only requested for the Bug Fix section of the `modules/cluster-logging-rn-5.6.adoc` file.

File: modules/cluster-logging-rn-5.6.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+//* (link:https://issues.redhat.com/browse/LOG-2913[LOG-2913])
+
+//* (link:https://issues.redhat.com/browse/LOG-2222[LOG-2222])
+
+[id="openshift-logging-5-6-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, if you deleted the Kibana Custom Resource, the {product-title} web console continued displaying a link to Kibana. With this update, removing the Kibana Custom Resource also removes that link. (link:https://issues.redhat.com/browse/LOG-2993[LOG-2993])
+
+* Before this update, a user was not able to view the application logs of namespaces they have access to. With this update, the Loki Operator automatically creates a cluster role and cluster role binding allowing users to read application logs. (link:https://issues.redhat.com/browse/LOG-3072[LOG-3072])
```
Comment (jc-berger) [inline modules/cluster-logging-rn-5.6.adoc:24]: In your previous bullet you use second person `if you deleted the Kibana Custom Resource` 

Here, you use `user`: `a user was not able to view the application` 

Be consistent with which tense you use. Since IBM SG encourages us to use second person, that's what I'd go with. 

Nevertheless, both versions (you/user) are correct. So, pick whichever version requires less time and editing on your end.

File: modules/cluster-logging-rn-5.6.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+* Before this update, the Operator removed any custom outputs defined in the `ClusterLogForwarder` custom resource when using LokiStack as the default log storage. With this update, the Operator merges custom outputs with the default outputs when processing the `ClusterLogForwarder` custom resource. (link:https://issues.redhat.com/browse/LOG-3090[LOG-3090])
+
+* Before this update, the CA key was used as volume name for mounting the CA into Loki, causing error states when the CA Key included non-conforming characters (such as dots). With this update, the volume name is standardized to an internal string which resolves the issue. (link:https://issues.redhat.com/browse/LOG-3331[LOG-3331])
```
Comment (jc-berger) [inline modules/cluster-logging-rn-5.6.adoc:28]: IBM SG doesn't want us using parenthesis for pretty much anything besides acronyms. 

I'd rephrase to simply, `...when the CA Key included non-conforming characters, such as dots.`

File: modules/cluster-logging-rn-5.6.adoc (hunk 1 above)
Comment (libander) [reply modules/cluster-logging-rn-5.6.adoc:24]: > Here, you use user: a user was not able to view the application -- here a "user" means not an "administrator" - you wouldn't be appropriate.

File: modules/cluster-logging-rn-5.6.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+//* (link:https://issues.redhat.com/browse/LOG-895[LOG-895])
+
+//* (link:https://issues.redhat.com/browse/LOG-2695[LOG-2695])
+
+//* (link:https://issues.redhat.com/browse/LOG-2715[LOG-2715])
+
+//* (link:https://issues.redhat.com/browse/LOG-2913[LOG-2913])
+
+//* (link:https://issues.redhat.com/browse/LOG-2222[LOG-2222])
+
```
Comment (anpingli) [inline modules/cluster-logging-rn-5.6.adoc:19]: Please add this EPIC https://issues.redhat.com/browse/LOG-3388

**Comment by anpingli** [comment]
@libander   I suggest adding the following known issues.
https://issues.redhat.com/browse/LOG-3482
https://issues.redhat.com/browse/LOG-3477
https://issues.redhat.com/browse/LOG-3470

**Comment by cbremble** [comment]
@libander for this known issue (only one we need to call out per @jcantrill ) LOG-3463 Jeff had included workaround info on the Jira ticket. It seems to me like the the wording makes it sound like a fix. I'll let you of course handle the verbiage but wanted to make sure that information was made available to you.

File: modules/cluster-logging-rn-5.6.adoc [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+* Before this update, a user was not able to view the application logs of namespaces they have access to. With this update, the Loki Operator automatically creates a cluster role and cluster role binding allowing users to read application logs. (link:https://issues.redhat.com/browse/LOG-3072[LOG-3072])
+
+* Before this update, the Operator removed any custom outputs defined in the `ClusterLogForwarder` custom resource when using LokiStack as the default log storage. With this update, the Operator merges custom outputs with the default outputs when processing the `ClusterLogForwarder` custom resource. (link:https://issues.redhat.com/browse/LOG-3090[LOG-3090])
+
+* Before this update, the CA key was used as volume name for mounting the CA into Loki, causing error states when the CA Key included non-conforming characters, such as dots. With this update, the volume name is standardized to an internal string which resolves the issue. (link:https://issues.redhat.com/browse/LOG-3331[LOG-3331])
```
Comment (rolfedh) [inline modules/cluster-logging-rn-5.6.adoc:40]:
```suggestion
* Before this update, the CA key was used as the volume name for mounting the CA into Loki, causing error states when the CA Key included non-conforming characters, such as dots. With this update, the volume name is standardized to an internal string which resolves the issue. (link:https://issues.redhat.com/browse/LOG-3331[LOG-3331])
```

File: modules/cluster-logging-rn-5.6.adoc [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+* Before this update, a default value set within the LokiStack Custom Resource Definition, cause an inability to create a LokiStack instance without a `ReplicationFactor` of 1. With this update,  the operator sets the actual value for the shirt size used. (link:https://issues.redhat.com/browse/LOG-3296[LOG-3296])
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-rn-5.6.adoc:42]: - "Shirt size" might be hard to some to understand. Sounds like jargon. Is there a better alternative?

**Review by rolfedh — APPROVED** [review]
Nice work. I've offered some suggestions. Hope this helps. Pre-approving to expedite.

File: modules/cluster-logging-rn-5.6.adoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (libander) [reply modules/cluster-logging-rn-5.6.adoc:42]: I think just 'size' works, thank you.

**Review by periklis — COMMENTED** [review]
Looks also good to me
